src/fusion/car_fuser.py

```diff
@@ -87,15 +80,15 @@ def _fill_car_like_components(self, points, ground_z, point_components,
 
             if valid_values.size != 0:
                 cc_z = np.mean(valid_values)
-                min_z = cc_z + self.min_height
-                max_z = cc_z + self.max_height
+                min_z = cc_z + min_height
+                max_z = cc_z + max_height
                 cluster_height = np.amax(points[cc_mask][:, 2])
                 if min_z <= cluster_height <= max_z:
-                    mbrect, _, mbr_width, mbr_length =\
+                    mbrect, _, mbr_width, mbr_length, _ =\
```

Comment:
In plaats van het berekenen van intersection percentages etc zouden we misschien ook gewoon kunnen kijken of de centerpoint van de mb_rect / cluster binnen een road_polygon valt.

Reply:
In beschrijving van Jira taakje